third_party/move/move-compiler/tests/move_check/parser/attribute_variants.exp

```diff
@@ -0,0 +1,60 @@
+error[E02015]: invalid attribute
```

Comment:
These too I think.

Reply:
I copied the same check to the /skip_attribute_checks/ directory where you can see the old result.